scala/src/com/google/idea/blaze/scala/run/BlazeScalaTestEventHandler.java

```diff
@@ -0,0 +1,37 @@
+package com.google.idea.blaze.scala.run;
+
+import com.google.idea.blaze.base.model.primitives.Kind;
+import com.google.idea.blaze.base.run.smrunner.BlazeTestEventsHandler;
+import com.google.idea.blaze.base.run.smrunner.SmRunnerUtils;
+import com.google.idea.blaze.scala.run.smrunner.BlazeScalaTestLocator;
```

Comment:
Did you forget to add this class?
It doesn't currently exist:
https://github.com/bazelbuild/intellij/tree/master/scala/src/com/google/idea/blaze/scala/run

Reply:
Oops, you're right, but it's not ready yet! I removed it for now...